SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/ReferenceCountedLockingPolicy/ReferenceCountedLockingPolicyTest.cpp

```diff
@@ -157,6 +161,28 @@ bool FAcquireLock::Update()
 	return true;
 }
 
+DEFINE_LATENT_AUTOMATION_COMMAND_THREE_PARAMETER(FAcquireLockViaDelegate, TSharedPtr<TestData>, Data, FName, ActorHandle, FString, DelegateLockIdentifier);
+bool FAcquireLockViaDelegate::Update()
+{
+	AActor* Actor = Data->TestActors[ActorHandle];
+
+	check(Data->AcquireLockDelegate.IsBound());
+	Data->AcquireLockDelegate.Execute(Actor, DelegateLockIdentifier);
+
+	return true;
```

Comment:
my mistake I know, but AcquireLockDelegate execution returns a bool which we can return instead?